.circleci/config.yml

```diff
@@ -8,6 +8,7 @@ jobs:
     branches:
       only:
         - master
+        - dev
```

Comment:
?! should it be in this PR?

Reply:
Because of @eickenberg's shenanigans. Will make sure it merges on top.